Makefile

```diff
@@ -123,12 +106,38 @@ load-db: login
 	docker-compose -f docker-compose.yml -f docker-compose.load-db.yml run load-db-dataset
 	docker-compose -f docker-compose.yml -f docker-compose.load-db.yml run load-db-entity
 
-cf-login:
-	cf target -o dluhc-digital-land || cf login -a api.london.cloud.service.gov.uk
+cf-check:
+# install dependencies
+ifndef CF_CLI
+ifeq ($(UNAME),Darwin)
+$(error CloudFoundry CLI not found in PATH)
+endif
+	curl https://packages.cloudfoundry.org/debian/cli.cloudfoundry.org.key | sudo apt-key add -
+	echo "deb https://packages.cloudfoundry.org/debian stable main" | sudo tee /etc/apt/sources.list.d/cloudfoundry-cli.list
+	sudo apt update
+	sudo apt install -y cf8-cli
```

Comment:
You'll want to switch this to v7, v8 doesn't support waiting for task completion when running migrations.